src/.rubocop.yml

```diff
@@ -12,6 +12,10 @@ Lint/StructNewOverride:
   Enabled: true
 Lint/RaiseException:
   Enabled: true
+Layout/SpaceAroundMethodCallOperator:
+  Enabled: true
+Style/ExponentialNotation:
+  Enabled: true
```

Comment:
this is already added https://github.com/emad-elsaid/rubyfunctions/blob/master/src/.rubocop.yml#L15